namesys/republisher/repub_test.go

```diff
@@ -40,7 +40,7 @@ func getMockNode(t *testing.T, ctx context.Context) *mockNode {
 	dstore := dssync.MutexWrap(ds.NewMapDatastore())
 	var idht *dht.IpfsDHT
 	h, err := libp2p.New(
-		libp2p.ListenAddrStrings("/ip4/127.0.0.1/tcp/0"),
+		libp2p.ListenAddrStrings("/ip4/0.0.0.0/tcp/0"),
```

Comment:
The tests were failing, and this fixed them. I guess some multiaddr private IP filtering could break it.

Reply:
Likely related to the new limit of connections per IP address in the Resource Manager. This was not the case until now. Another way of addressing this would be using `WithLimitPeersPerCIDR` I think.

_No action needed. Just informative._